packages/admin-panel/src/routes/visualisations/dashboardRelations.js

```diff
@@ -15,6 +15,7 @@ export const FIELDS = {
       optionLabelKey: 'dashboard.code',
       optionValueKey: 'dashboard.id',
       sourceKey: 'dashboard_id',
+      accessor: record => record['dashboard.code'],
```

Comment:
If an accessor is not set, the source key is used to get the initial data. Since we are trying to display the code, nothing was showing up.